server.js

```diff
@@ -2,7 +2,7 @@ import express from 'express'
 import http from 'http'
 import expressStaticGzip from 'express-static-gzip'
 
-const port = '4200'
+const port = '420O'
```

Comment:
**Suggestion:** Correct the port number. It should be a number instead of a string with a letter.
```suggestion
const port = 4200
```